modules/pubmaticBidAdapter.js

```diff
@@ -182,8 +184,11 @@ let biddersList = ['pubmatic'];
 const allBiddersList = ['all'];
 
 const vsgDomain = window.location.hostname;
```

Comment:
first-party data is partitioned at the domain level, why do we need vsgDomain as key?